pkg/systemd/hwinfo.jsx

```diff
@@ -43,90 +43,87 @@ import { Switch } from "@patternfly/react-core/dist/esm/components/Switch/index.
-                            </DescriptionListGroup>
-                            { onSecurityClick !== undefined && <DescriptionListGroup>
-                                <DescriptionListTerm>{ _("CPU security") }</DescriptionListTerm>
-                                <DescriptionListDescription>{ mitigations }</DescriptionListDescription>
-                            </DescriptionListGroup>}
-                        </> }
-                    </DescriptionList>
-                </FlexItem>
-            </Flex>
-        );
-    }
-}
+    return (
+        <Flex id="hwinfo-system-info-list" direction={{ default: 'column', sm: 'row' }}>
+            <FlexItem className="hwinfo-system-info-list-item" flex={{ default: 'flex_1' }}>
+                <DescriptionList className="pf-m-horizontal-on-md">
+                    { info.type &&
+                        <DescriptionListGroup>
+                            <DescriptionListTerm>{ _("Type") }</DescriptionListTerm>
+                            <DescriptionListDescription>{ info.type }</DescriptionListDescription>
+                        </DescriptionListGroup> }
+                    { info.name &&
+                        <DescriptionListGroup>
+                            <DescriptionListTerm>{ _("Name") }</DescriptionListTerm>
+                            <DescriptionListDescription>{ info.name }</DescriptionListDescription>
+                        </DescriptionListGroup> }
+                    { info.version &&
+                        <DescriptionListGroup>
+                            <DescriptionListTerm>{ _("Version") }</DescriptionListTerm>
+                            <DescriptionListDescription>{ info.version }</DescriptionListDescription>
+                        </DescriptionListGroup> }
+                </DescriptionList>
+            </FlexItem>
+            <FlexItem className="hwinfo-system-info-list-item" flex={{ default: 'flex_1' }}>
+                <DescriptionList className="pf-m-horizontal-on-md">
+                    { info.bios_vendor && <>
+                        <DescriptionListGroup>
+                            <DescriptionListTerm>{ _("BIOS") }</DescriptionListTerm>
+                            <DescriptionListDescription>{ info.bios_vendor }</DescriptionListDescription>
+                        </DescriptionListGroup>
+                        <DescriptionListGroup>
+                            <DescriptionListTerm>{ _("BIOS version") }</DescriptionListTerm>
+                            <DescriptionListDescription>{ info.bios_version }</DescriptionListDescription>
+                        </DescriptionListGroup>
+                        <DescriptionListGroup>
+                            <DescriptionListTerm>{ _("BIOS date") }</DescriptionListTerm>
+                            <DescriptionListDescription>{ bios_date ? timeformat.date(bios_date) : info.bios_date }</DescriptionListDescription>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19378-20230925-080729-1e17ab19-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)